### ISSUE TITLE
New UI - The Slider

### ISSUE THREAD

**Issue by jstandard1** [jstandard1/goodrecs#190]
- [Specs Directory](www.example.com)

**Basic Description**
A method which allows the user to very quickly move through the list of menu items.

**Requirements**

1. The user can very quickly scroll through the list of menu items. (ex. Moving from Item 1 to Item N much more quickly than regular, swipe-based navigation)
2. The user will be warned with a sound once they arrive at the last item in the list. This happens if they are navigating the list from Left to Right and from Right to Left.
3. The user can stop the scrolling at any point and the list will end scrolling on that item.
4. If the user stops the scrolling when the menu is between two menu items, the menu will snap onto the item closest to the center of the screen.
5. If the user tries to tap/gesture on the screen while using the quick navigation, the screen will not respond to the tap/gesture command,

**Major Dependencies**
List of features/Trello cards that must be completed first

**Out of Scope**

- This only applies to Menu Item Lists and DOES NOT apply to the Global Nav
